src/backend/joanie/core/utils/contract.py

```diff
@@ -18,8 +18,10 @@ def _get_base_signature_backend_references(
     course_product_relation=None, organization=None, extra_filters=None
 ):
     """
-    Build the base query to get signature backend references from either a Course Product Relation
-    object or an Organization object when the contract is signed.
+    Build the base query to get signature backend references from a Course Product Relation
+    object or an Organization object when the contract is signed. You can parse both parameters if
```

Comment:
```suggestion
    object or an Organization object when the contract is signed. You can pass both parameters if
```

Reply:
I've changed all "parsing" to "passing" on the scope of this PR.